packages/expo-router/src/color/index.ts

```diff
@@ -112,6 +112,50 @@ const androidColor = new Proxy(
   }
 );
 
+/**
+ * Color utility to access platform-specific colors easily.
+ *
+ * On **Android**, it provides access to:
+ * - System colors, as a type-safe wrapper over `PlatformColor`. For example, `Color.android.background`.
+ * - Attribute colors, as a type-safe wrapper over `PlatformColor`. For example, `Color.android.attr.colorPrimary`.
+ * - [Material Design 3 static colors](https://m3.material.io/styles/color/static/baseline). For example, `Color.android.material.primary`.
+ * - [Material Design 3 dynamic colors](https://m3.material.io/styles/color/dynamic/user-generated-source). For example, `Color.android.dynamic.primary`.
+ *
+ * On **iOS**, it is a type-safe wrapper over `PlatformColor`, providing access to system colors. For example, `Color.ios.label`.
+ *
+ * **Note**: To ensure the colors align with the system theme on Android, make sure they are used within a component that responds to theme changes, such as by using the `useColorScheme` hook from React Native.
```

Comment:
```suggestion
 * > **Note**: To ensure the colors align with the system theme on Android, make sure they are used within a component that responds to theme changes, such as by using the `useColorScheme` hook from React Native.
```

Let's use a callout here.